client/components/Modal.tsx

```diff
@@ -5,11 +5,20 @@ import { Checkbox } from ".";
 type Props = {
   children: ReactNode;
   isOpen?: boolean;
```

Comment:
no need for this to be optional, as we are always passing a value. This should be a mandatory field, else the modal wouldnt stand a hope to ever open